api/actions/views.py

```diff
@@ -0,0 +1,191 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.shortcuts import get_object_or_404
+from rest_framework import generics
+from rest_framework import permissions
+from rest_framework.exceptions import PermissionDenied
+
+from framework.auth.oauth_scopes import CoreScopes
+from osf.models import Action
+from reviews import permissions as reviews_permissions
+
+from api.actions.serializers import ActionSerializer
+from api.base.exceptions import Conflict
+from api.base.parsers import (
+    JSONAPIMultipleRelationshipsParser,
+    JSONAPIMultipleRelationshipsParserForRegularJSON,
+)
+from api.base.utils import absolute_reverse
+from api.base.views import JSONAPIBaseView
+from api.base import permissions as base_permissions
+
+
+class ActionMixin:
```

Comment:
It was a function originally, but quoth @chrisseto:
> I'd vote for a mixin here rather than having a functions just floating around.

But he quit so function it is!